# Standard ROS header
Header header

# number of points
uint32 num_points

# X points
float32[] px
# Y points
float32[] py